src/tools/muon.ts

```diff
@@ -42,7 +42,7 @@ export async function lint(muon: Tool, root: string, document: vscode.TextDocume
   return diagnostics;
 }
 
-export async function format(muon: Tool, document: vscode.TextDocument): Promise<vscode.TextEdit[]> {
+export async function format(muon: Tool, root: string, document: vscode.TextDocument): Promise<vscode.TextEdit[]> {
```

Comment:
Wanna call this argument sourceRoot?

Reply:
Only if `lint()` is changed accordingly...